src/cluster.c

```diff
@@ -134,10 +138,15 @@ int verifyDumpPayload(unsigned char *p, size_t len, uint16_t *rdbver_ptr) {
     if (server.skip_checksum_validation)
         return C_OK;
 
+    uint64_t crc_payload;
+    memcpy(&crc_payload, footer+2, 8);
```

Comment:
should we concern about endian issue

Reply:
crc_payload is encoded by `memrev64ifbe`, and crc also is encoded later, so this is correct, right?

Reply:
yes, it should be correct.